crates/server/src/services/transparency/map.rs

```diff
@@ -49,7 +49,8 @@ pub fn spawn(input: Input) -> Output {
                 summary = log_summary_rx.recv() => {
                     if let Some(summary) = summary {
                         let leaf = summary.leaf;
-                        map = map.insert(leaf.log_id.clone(), MapLeaf { record_id: leaf.record_id.clone() });
+                        let hash = leaf.log_id.0.clone().try_into().unwrap();
```

Comment:
Likewise, I don't think this conversion is needed and we can continue to just insert `leaf.log_id.clone()`, unless I'm mistaken?